interfaces/mount/change.go

```diff
@@ -42,6 +44,29 @@ type Change struct {
 	Action Action
 }
 
+// Perform executes the desired mount or unmount change.
```

Comment:
Thanks for the review. My main goal here was to get some comments on the mount syscall vs mount command (and libmount). The particular details of error handling are not that important. They are just consistent with go and in this specific case I don't believe it is appropriate to panic.